spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'Logins', type: :feature do
+  background do
+    visit root_path
+  end
+
+  scenario 'show groups_path' do
```

Comment:
- [ ] You did a great job. However, you have some failing tests that need to be fixed. Kindly fix these tests to make your project approve.

<img width="682" alt="Screen Shot 2022-10-14 at 20 14 50" src="https://user-images.githubusercontent.com/69011963/195883744-31c6adf8-e156-486a-8762-220371950d7a.png">